src/Platform/Microsoft.Testing.Platform/Messages/TestNodeProperties.cs

```diff
@@ -314,7 +314,7 @@ public sealed record TestFileLocationProperty(string FilePath, LinePositionSpan
 /// </summary>
 /// <param name="AssemblyFullName">Assembly full name.</param>
 /// <param name="Namespace">Namespace.</param>
-/// <param name="TypeName">Type name.</param>
+/// <param name="TypeName">Type name in metadata format, not including the namespace. Generics are represented by backtick followed by arity. Nested types are represented by <c>+</c>.</param>
```

Comment:
Please add a link to the spec

Reply:
I want to avoid linking to VSTest stuff. The current documentation for TypeName seems reasonable to me though.